src/components/Incorporation/IncorporationDateTime.vue

```diff
@@ -18,7 +18,6 @@
       >
         <v-radio-group
           v-model="selectDate"
-          column
```

Comment:
This is one of the things that have been removed and need to find an alternative for. 
![IncorporationDateTime](https://github.com/bcgov/business-create-ui/assets/122301442/96b19a66-3ff3-4087-b34e-74d10e41d0b8)

Need to do that once the project runs properly so we can see what things actually look like.

Ditto line 42 here.

Reply:
https://vuetifyjs.com/en/api/v-radio-group/#props-direction

Reply:
You fixed it. Resolving.